src/api-service/__app__/onefuzzlib/azure/monitor.py

```diff
@@ -4,17 +4,17 @@
 # Licensed under the MIT License.
 
 import os
-from typing import Any, Dict
+from typing import Dict
 
 from azure.mgmt.loganalytics import LogAnalyticsManagementClient
 from memoization import cached
 
-from .creds import get_base_resource_group, mgmt_client_factory
+from .creds import get_base_resource_group, get_identity, get_subscription
 
 
-@cached(ttl=60)
-def get_montior_client() -> Any:
-    return mgmt_client_factory(LogAnalyticsManagementClient)
+@cached
```

Comment:
Why the TTL change? Why here and not below?

Reply:
We cache the client forever.  It handles renewing auth tokens automatically.

However, the settings for Azure Monitor could change, so we keep those such that they expire ever so often.